.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature request
+assignees: ''
+
+---
+
+Note: Please only use this template to make feature requests of the `chatkit-client-js` SDK. For general queries dedicated support is always available at https://support.pusher.com and support@pusher.com
```

Comment:
I  would be even more explicit that it should be a feature request directly pertaining to the SDK, and not a feature request for the product as a whole.

Reply:
Do you have those words?..